pkg/unshare/unshare.go

```diff
@@ -195,13 +195,25 @@ func (c *Cmd) Start() error {
 				if err == nil {
 					gidmapSet = true
 				} else {
-					fmt.Fprintf(continueWrite, "error running newgidmap: %v: %s", err, g.String())
-					fmt.Fprintf(continueWrite, "falling back to single mapping\n")
+					logrus.Warnf("error running newgidmap: %v: %s", err, g.String())
```

Comment:
I don't think these messages show to the user by default?  Do the messages still show on the terminal?

Reply:
The logrus.Warnf messages only appear to the user if the --debug flag is on. Is that how you think this should function? When writing to continueWrite before the command would always exit with status code 1.

Reply:
@nalind thoughts on why the Fprintf to continueWrite would exit out here?

Reply:
[The child](https://github.com/containers/buildah/blob/fa67ecb27af4ca6faad5509d9837e05c32ed63cb/pkg/unshare/unshare.c#L229) treats any data that it reads from the reading end of the pipe as an indication that a fatal error has occurred and that it needs to exit.  Messages that aren't about fatal errors should not be written to the pipe.